### PR TITLE
fix sudo -iu ebuser eb --job

### PR DESCRIPTION
**Pull request by mboisson**
The `SBATCH_OUTPUT` still is not honoured however...  (see EasyBuild's Slack)

### CODE REVIEW

File: eb [hunk 1]
```diff
@@ -127,7 +134,7 @@ export RSNT_EASYBUILD_MAGIC_COOKIE=263ca73bb634185aab1d1b41627fdbba
 NEW_ARGS=()
 for argument in "$@"; do
 	if [[ "$CURRENT_USER" == "ebuser" ]]; then
-		if [[ $argument =~ --inject-checksums* || $argument =~ --robot* || "$argument" == "-r" ]]; then
+		if [[ $argument =~ --inject-checksums* || $argument =~ --robot$ || "$argument" == "-r" ]]; then
```
Comment (bartoldeman) [inline eb:137]: why should this only apply to `--robot` at EOL?

File: eb [hunk 2]
```diff
@@ -13,7 +13,11 @@ if [[ $HOSTNAME =~ archimedes.c3.ca$ ]]; then
 	export PATH=$PATH:/opt/software/slurm/bin
 	export EASYBUILD_JOB_BACKEND='Slurm'
 	export SBATCH_MEM_PER_CPU='3500m'
+	if [[ $CURRENT_USER == "ebuser" ]]; then
+		export SBATCH_OUTPUT="/cvmfs/local/var/log/slurm-%x-%j.out"
+	fi
```
Comment (bartoldeman) [inline eb:18]: I'm afraid we either need to change framework or look into g3cpie? The framework sets `--output=<name>-%j.out` explicitly.

**Review by bartoldeman — CHANGES_REQUESTED** [review]

File: eb (hunk 2 above)
Comment (mboisson) [reply eb:18]: Yeah, that's what I conclude too with Kenneth's answer on EB slack

File: eb (hunk 1 above)
Comment (mboisson) [reply eb:137]: It sends `--robot-path` (but without `--robot`) to the job script

**Comment by mboisson** [comment]
lgtm

**Pull request merged**